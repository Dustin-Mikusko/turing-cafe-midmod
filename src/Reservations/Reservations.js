import React from 'react';
import './Reservations.css';

const Reservations = ({ reservations }) => {
  const reservations = reservations.map(resy => {
    return <Reservation 
      key={resy.id}
      name={resy.name}
      date={resy.date}
      time={resy.time}
      number={resy.number}
    />
  });

  return (
    <section>
      {reservations}
    </section>
  )
}

export default Reservations;
