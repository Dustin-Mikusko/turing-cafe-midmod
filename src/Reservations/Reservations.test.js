import React from 'react';
import { shallow } from 'enzyme';
import Reservations from './Reservations';

describe('Reservations', () => {
  it('should render component with correct data to match snapshot', () => {
    let mockReservations = [
      {
        id: 1,
        name: 'Dustin',
        date: '7/13',
        time: '6:30',
        number: 4,
      }
    ]
    const wrapper = shallow(<Reservations reservations={mockReservations}  />);

    expect(wrapper).toMatchSnapshot();
  })
})
