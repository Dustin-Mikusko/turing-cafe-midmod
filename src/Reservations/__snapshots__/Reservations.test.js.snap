// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Reservations should render component with correct data to match snapshot 1`] = `
<section
  className="resys-container"
>
  <Reservation
    date="7/13"
    key="1"
    name="Dustin"
    number={4}
    time="6:30"
  />
</section>
`;
